cms/articles/wagtail_hooks.py

```diff
@@ -0,0 +1,24 @@
+from typing import TYPE_CHECKING
+
+from wagtail import hooks
+from wagtail.admin import messages
+
+from cms.articles.models import StatisticalArticlePage
+
+if TYPE_CHECKING:
+    from django.http import HttpRequest
+    from wagtail.models import Page
+
+
+@hooks.register("before_create_page")
+def before_create_page(
+    request: "HttpRequest",
+    parent_page: "Page",
+    page_class: type["Page"],
+) -> None:
+    if page_class == StatisticalArticlePage and parent_page.get_latest():
```

Comment:
note: it would be more efficient to have a `has_articles` which checks that the series has children.

It is not unthinkable to have a draft article in the series that adds the headline figure the editors want to link to.

Reply:
We pre-populate the new article with the latest published article, and the current implementation also only allows to pick headline figures from published articles.

If we want to allow editors to choose headline figures from drafts, we would need to refactor the code to allow that, but also then decide here if, when creating a new article, do we want to pre-populate it with content from the latest published article, or the most recently created article, even if it's a draft.